package.json

```diff
@@ -4,13 +4,26 @@
   "license": "MIT",
   "type": "module",
   "author": "Thinkmill Labs Pty Ltd",
-  "repository": "https://github.com/thinkmill/manylicenses",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/thinkmill/manylicenses.git"
+  },
```

Comment:
npm asks for this thusly:
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository